ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/individual-journey/pages/about-your-computer/about-your-computer.component.html

```diff
@@ -1,5 +1,5 @@
 <div class="govuk-grid-row">
-  <div class="govuk-grid-column-two-thirds">
+  <div class="govuk-grid-column-full">
```

Comment:
I've been thinking of this, I've seen it in other pages as well. Are we supposed to use the full width one? In which case, do you think you could double check the other pages because they might be wrong?

Reply:
I have found one on about-you, it changed to full column